src/test/java/teammates/test/cases/ui/pagedata/InstructorFeedbackRemindParticularStudentsPageDataTest.java

```diff
@@ -0,0 +1,49 @@
+package teammates.test.cases.ui.pagedata;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertNull;
+
+import java.util.ArrayList;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.FeedbackSessionResponseStatus;
+import teammates.test.cases.BaseComponentTestCase;
+import teammates.ui.controller.InstructorFeedbackRemindParticularStudentsPageData;
+import teammates.ui.template.RemindParticularStudentsCheckboxEmailNamePair;
+
+public class InstructorFeedbackRemindParticularStudentsPageDataTest extends BaseComponentTestCase {
```

Comment:
The test should be capable of passing without extending from `BaseComponentTestCase`.
Extend from `BaseTestCase` or nothing else instead.